TransferMethod/Sources/View/Widget/AbstractWidget.swift

```diff
@@ -80,7 +88,26 @@ class AbstractWidget: UIStackView, UITextFieldDelegate {
     }
 
     func isValid() -> Bool {
-        return !isInvalidEmptyValue() && !isInvalidLength() && !isInvalidRegex()
+        if isInvalidEmptyValue() || isInvalidLength() || isInvalidRegex() {
```

Comment:
probably create 1 var isValid = true instead of returning true and false. That's more cleaner

Reply:
👍